R/pkg/R/DataFrame.R

```diff
@@ -2287,16 +2287,19 @@ setMethod("mutate",
 
             # For named arguments, use the names for arguments as the column names
             # For unnamed arguments, use the argument symbols as the column names
-            args <- sapply(substitute(list(...))[-1], deparse)
```

Comment:
R 4.0.0 adds `deparse1` that would have been more appropriate here:

> `deparse1()` is a simple utility added in R 4.0.0 to ensure a string result (character vector of length one), typically used in name construction, as `deparse1(substitute(.))`.

That function is just a wrapper so easy to backport:

```
deparse1 = function (expr, collapse = " ", width.cutoff = 500L, ...) 
paste(deparse(expr, width.cutoff, ...), collapse = collapse)
```

(though personally I would still stick with `trimws`)

Reply:
no longer relevant